### PR TITLE
Make home page widgets only draggable via handle

### PR DESCRIPTION
**Pull request by kmshin1397**
drag

### CODE REVIEW

**Comment by dmitryduev** [comment]
The code looks good, thanks @kmshin1397! 
The top right corner looks a bit messy with this bc the icons are not aligned (even though I guess it's intentional). Could you please try implementing something like
image
(as always, if not too much trouble)

File: static/js/components/HomePage.jsx [hunk 1]
```diff
@@ -87,17 +89,31 @@ const HomePage = () => {
       cols={{ xlg: 16, lg: 12, md: 10, sm: 6, xs: 4 }}
       margin={[15, 15]}
       onLayoutChange={LayoutChangeHandler}
+      draggableHandle=".dragHandle"
     >
       <div key="sourceList" className={styles.homePageWidgetDiv}>
+        <DragHandleIcon className={`${styles.dragHandle} dragHandle`} />
```
Comment (dannygoldstein) [inline static/js/components/HomePage.jsx:95]: Can these `DragHandleIcon`s be incorporated directly into the corresponding components?

**Comment by kmshin1397** [comment]
Refactored a bit so that the Home Page defines general styles to pass along to child widgets, who use them when making the Paper, drag handler, and other icons (i.e. preferences).

Screenshot 2020-09-01 190415

**Comment by dmitryduev** [comment]
Looks awesome, thanks! Merging.

**Pull request merged**